redis-om-spring/src/main/java/com/redis/om/spring/RedisModulesConfiguration.java

```diff
@@ -284,43 +285,51 @@ private List<Field> findIndexFields(java.lang.reflect.Field field, String prefix
 
       Indexed indexed = (Indexed) field.getAnnotation(Indexed.class);
 
+      Class<?> fieldType = ClassUtils.resolvePrimitiveIfNecessary(field.getType());
```

Comment:
do these changes pertain to the actual crux of this issue we're resolving? or is this just some code cleanup?

Reply:
Moved to a separate branch